cdap-integration-test/src/main/java/co/cask/cdap/test/remote/RemoteApplicationManager.java

```diff
@@ -127,25 +77,7 @@ public MapReduceManager startMapReduce(final String programName) {
 
   @Override
   public MapReduceManager startMapReduce(final String programName, Map<String, String> arguments) {
```

Comment:
We do, I guess it's the result isn't stored in a temp variable; that's the only difference.